### PR TITLE
tidb-config: add value range for token-limit

### PR DESCRIPTION
**Pull request by TomShawn**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add the value range for `token-limit` in the TiDB Configuration File document.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tidb/issues/24226

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-configuration-file.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ The TiDB configuration file supports more options than command-line parameters.
 
 + The number of sessions that can execute requests concurrently.
 + Default value: `1000`
++ Value range: \> `0`
```
Comment (XuHuaiyu) [inline tidb-configuration-file.md:24]: Will `(0, +∞)` be better?

Comment (TomShawn) [reply tidb-configuration-file.md]: Suggestion applied. PTAL again. Thanks!

File: tidb-configuration-file.md [hunk 2]
```diff
@@ -21,6 +21,7 @@ The TiDB configuration file supports more options than command-line parameters.
 
 + The number of sessions that can execute requests concurrently.
 + Default value: `1000`
++ Value range: (`0`, `+∞`)
```
Comment (yikeke) [inline tidb-configuration-file.md:24]:
```suggestion
+ Type: Integer
+ Default value: `1000`
+ Minimum value: `1`
+ Maximum value: 
```

**Review by yikeke — APPROVED** [review]

**Pull request merged**